tests/functional/test_auth_sigv4.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is there a reason we are not putting this in the unit test suite under `auth/test_sigv4.py`?

Reply:
I added a new file because the other test file seems specific to the tests generated from the aws4_testsuite directory. I could go either way on this.

Reply:
I think we should be put it in the`unit` test directory. Not too picky on which file, I just suggested that one because it looked similar to some of the lines that you had.